lib/libraries/c3/extensions/default-date-format.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function(startDate, endDate){
```

Comment:
This is more like a "helper" than a c3 extension.. c3 doesn't support the concept of an "extension" anyway. Will probably move this into a more explicit directory at some point :+1: